doc/releases/octopus.rst

```diff
@@ -1,3 +1,17 @@
+v15.2.6 Octopus
+===============
+
+This is the 6th backport release in the Octopus series. This releases fixes a
+security flaw affecting Messenger V2 for Octopus & Nautilus. We recommend users
+to update to this release.
+
+Notable Changes
+---------------
+
+* CVE 2020-25660: Fix a regression in Messenger V2 replay attacks
```

Comment:
Similarly, "CVE 2020-25660: CEPHX_V2 replay attack protection lost (both for Messenger v1 and Messenger v2)"